modules/jaeger-deploy-production-es.adoc

```diff
@@ -0,0 +1,143 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-deploying.adoc
+////
+
+[id="jaeger-deploy-production_{context}"]
+= Deploying the Jaeger production strategy
+
+The `production` deployment strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required.
+
+.Prerequisites
+
+* The Elasticsearch Operator must be installed.
+* An Elasticsearch instance must be deployed.
```

Comment:
This is not a requirement - the Jaeger Operator (if ES operator is installed and es.server-urls not defined) will self provision the ES instance.